can-construct_test.js

```diff
@@ -159,3 +159,44 @@ test("setup called with original arguments", function(){
 
 	Construct.extend(o1, o2);
 });
+
+test("legacy namespace strings (A.B.C) accepted", function() {
+
+	var Type = Construct.extend("Foo.Bar.Baz");
+
+	ok(new Type() instanceof Construct, "No unexpected behavior in the prototype chain");
+	equal(Type.name, "Foo_Bar_Baz", "Name becomes underscored");
```

Comment:
@bmomberger-bitovi it would make sense not to use `eval`  It's been suggested here (https://github.com/canjs/can-construct/issues/12 ... which links to the original discussion).  Anyway you could combine these?